modules/mono/glue/GodotSharp/GodotPlugins/Main.cs

```diff
@@ -123,7 +123,7 @@ internal void Unload()
             }
             catch (Exception e)
             {
-                Console.Error.WriteLine(e);
+                WriteLineError(e);
```

Comment:
`NativeFuncs` may not be initialized here, the exception could be thrown before the `NativeFuncs.Initialize` call or in the middle of it.